app/helpers/error_helper.rb

```diff
@@ -31,11 +31,16 @@ def self.missing_required_metadata(sample, missing_metadata_fields)
     end
   end
 
+  LOCATION_INVALID_ERROR = "Please input a location.".freeze
   DATE_INVALID_ERROR_HUMAN = "Please input a date in the format YYYY-MM or MM/YYYY. (for human samples)".freeze
   DATE_INVALID_ERROR = "Please input a date in the format YYYY-MM-DD, YYYY-MM, MM/DD/YY, or MM/YYYY.".freeze
   NUMBER_INVALID_ERROR = "Please input a number.".freeze
 
   def get_field_error(field, is_human = false)
+    if field.base_type == 3
```

Comment:
There are not constants to use instead of this magic numbers?

Reply:
fixed 👍